src/node.cc

```diff
@@ -4716,6 +4705,7 @@ int Initialize(int argc, const char** argv, const bool evaluate_stdin) {
 
   // Hack around with the argv pointer. Used for process.title = "blah --args".
   // argv won't be modified
+  // TODO(cmfcmf): This returns argv, which therefore can't be const!
   uv_setup_args(argc, const_cast<char**>(argv));
```

Comment:
The original code in the node master branch looks like this:
https://github.com/nodejs/node/blob/8ed0f3409a31abae87bf6aae394b8901c5e6ecab/src/node.cc#L4582-L4583